spec/factories/campaigns.rb

```diff
@@ -19,10 +19,19 @@
 FactoryGirl.define do
   factory(:campaign) do
     volunteer_center
-    name 'MyString'
-    description 'MyString'
-    donation_dateline Date.today + 7
+    name 'Share Your Christmas 2015'
+    description 'Share Your Christmas Campaign 2015'
+    donation_deadline Date.today + 7
     reminder_date Date.today + 14
-    logo { fixture_file_upload(Rails.root.join('public', 'images', 'original', 'missing.png')) }
+    # logo { fixture_file_upload(Rails.root.join('public', 'images', 'original', 'missing.png')) }
```

Comment:
If this is breaking let me know and I'll take a look at it. Might be able to use a basic File path